raiden-ts/src/raiden.ts

```diff
@@ -326,7 +327,9 @@ export class Raiden {
       raidenReducer,
       // workaround for redux's PreloadedState issues with branded values
       state as any, // eslint-disable-line @typescript-eslint/no-explicit-any
-      applyMiddleware(loggerMiddleware, persisterMiddleware, this.epicMiddleware),
+      composeWithDevTools(
```

Comment:
I'd rather have it applied only with development builds. Any chance of doing something like this?:
```ts
let middlewares = applyMiddleware<...>(loggerMiddleware, ...);
if (process.env.NODE_ENV !== 'production')
  middlewares = composeWithDevTools<...>(middlewares);
this.store = createStore(
  ...,
  middlewares,
);
```
You may need to specify the generics explicitly. Just hover at the current objects in order to check them out.

Reply:
The `composeWithDevTools` function should to do this already. It uses the default `compose` of Redux in development environments. See the source code: https://github.com/zalmoxisus/redux-devtools-extension/blob/c220fd18d6fb4b0790a501ec4df0de52b5bee460/npm-package/developmentOnly.js#L7
Sorry for not mentioning.